### DISCUSSION TITLE
Instancia Mastodon oficial de BcnEng

### DISCUSSION THREAD

**Discussion by akira28** [bcneng/website#133] [Ideas]
Probablemente es demasiado curro, pero sería muy guay tener una instancia de Mastodon de BcnEng. Lo nuevo de [Cloudflare](https://blog.cloudflare.com/welcome-to-wildebeest-the-fediverse-on-cloudflare/) simplifica bastante las cosas. WDYT? Ofrezco algo de mi tiempo libre!

**Comment by smoya** [comment]
Something to consider as well https://masto.host/pricing/
